openpype/hosts/houdini/plugins/load/load_bgeo.py

```diff
@@ -94,9 +93,10 @@ def update(self, container, representation):
 
         # Update the file path
         file_path = get_representation_path(representation)
-        file_path = self.format_path(file_path)
+        is_sequence = bool(representation["context"].get("frame"))
+        file_path = self.format_path(file_path, is_sequence)
```

Comment:
How about changing `format_path()` arguments to `(file_path, representation)`. The `is_sequence = bool(representation["context"].get("frame")` statement would then just move into the method.

Thoughts?

Reply:
sure, it's much more cleaner.